### PR TITLE
`Assessment`: Show legend warning element only for tutors when assessing example submissions for modeling exercises

### PR DESCRIPTION
**Pull request by Olimbia23**
#### General

- [x] I tested **all** changes and their related features with **all** corresponding user types on a test server.
- [x] This is a small issue that I tested locally and was confirmed by another developer on a test server.
- [ ] * Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.ase.in.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/development-process.html#naming-conventions-for-github-pull-requests).
#### Server
- [ ] * I followed the [coding and design guidelines](https://docs.artemis.ase.in.tum.de/dev/guidelines/server/).
- [ ] * I added multiple integration tests (Spring) related to the features (with a high test coverage).
- [ ] * I added `@PreAuthorize` and checked the course groups for all new REST Calls (security).
- [ ] * I implemented the changes with a good performance and prevented too many database calls.
- [ ] * I documented the Java code using JavaDoc style.
#### Client
- [x] I followed the [coding and design guidelines](https://docs.artemis.ase.in.tum.de/dev/guidelines/client/).
- [ ] * I added multiple integration tests (Jest) related to the features (with a high test coverage), while following the [test guidelines](https://docs.artemis.ase.in.tum.de/dev/guidelines/client-tests/).
- [ ] * I added `authorities` to all new routes and checked the course groups for displaying navigation elements (links, buttons).
- [ ] * I documented the TypeScript code using JSDoc style.
- [x] I added multiple screenshots/screencasts of my UI changes.
- [ ] * I translated all newly inserted strings into English and German.
#### Changes affecting Programming Exercises
- [ ] * I tested **all** changes and their related features with **all** corresponding user types on Test Server 1 (Atlassian Suite).
- [ ] * I tested **all** changes and their related features with **all** corresponding user types on Test Server 2 (Jenkins and Gitlab).

[ ] * - not needed

### Motivation and Context

Legend view was introduced for example submissions modeling editor. Right now for both instructors and tutors that are reading or assessing the example submissions the "Incorrect assessment" legend element is shown.
This PR makes it to be shown only for tutors that are assessing the example submission as it only makes sense to show during the training process.

### Description

The legend element is shown only for `toComplete` being set to true (which means that the example submission needs to be assessed)

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Tutor

1. Log in to Artemis as an Instructor
2. Navigate to Course Administration
3. Navigate to Modeling Exercise or Create your own
4. Create an Example Submission and example assessment and don't check Used For Tutorial. 
5. Create another Example Submission and example assessment and check Used For Tutorial. 
6. The legend element "Incorrect assessment" **should not be shown** when creating the example assessment.
---

1. Log in to Artemis as Tutor
2. Navigate to Course Administration
3. Navigate to Modeling Exercise Assessment Dashboard
4. Read an Example Submission -> the legend element **should not be shown**
5. Assess an Example Submission -> the legend element **should be shown**

#### Code Review
- [x] Review 1
- [x] Review 2
#### Manual Tests
- [x] Test 1
- [x] Test 2

#### Read
Screen Shot 2021-11-22 at 00 13 56

#### Assess
Screen Shot 2021-11-22 at 00 14 16

### CODE REVIEW

**Review by lschlesinger — COMMENTED** [review]
I think the term "Review example submission"  in guidedTourTutorAssessment.json should also be refactored, right? you should search all files again for `reviewSubmission`.
And in the dashboard, there is the word review in the banner (see picture). I think you need to one more check the whole application for using consistent terms review/read when referring to interacting with the example submission.
Bildschirmfoto 2021-11-22 um 16 31 57

I also tested on TS1 if the icon appears / disappears in the described situation, it worked fine.

**Review by MerlinMehmed — COMMENTED** [review]
Tested on TS3, works as expected, code changes also look good!

**Review by MerlinMehmed — APPROVED** [review]
I forgot to press the Aprrove option the previous time :D

**Review by lschlesinger — DISMISSED** [review]
Approve after adapting the other translations! 👍

**Review by MerlinMehmed — APPROVED** [review]
Reapprove

**Review by filip-gregurevic — APPROVED** [review]
re-approve

**Review by VinnavanSembian — APPROVED** [review]
Tested as described in the testing steps. Everything works as expected.

**Review by cremertim — APPROVED** [review]
Tested during testing session, everything works as described in the testing steps. 
Code changes look good to me

**Review by lBruchmann — APPROVED** [review]
Tested during testing session. Tested on ts3. I followed the testing steps. Everything works as expected.

File: src/main/webapp/i18n/de/exampleSubmission.json [hunk 1]
```diff
@@ -23,14 +23,10 @@
...
-            "assessScore": {
-                "success": "Herzlichen Glückwunsch! Das ist eine gute Beurteilung!",
-                "tooLow": "Die Punktzahl deiner Bewertung dieser Abgabe ist zu niedrig. Bitte überprüfe die Bewertungskriterien und die Abgabe noch einmal sorgfältig und reiche deine verbesserte Bewertung ein.",
-                "tooHigh": "Die Punktzahl deiner Bewertung dieser Abgabe ist zu hoch. Bitte überprüfe die Bewertungskriterien und die Abgabe noch einmal sorgfältig und reiche deine verbesserte Bewertung ein."
-            },
+            "correctTutorAssessment": "Herzlichen Glückwunsch! Das ist eine gute Beurteilung!",
             "submitSuccessful": "Deine Beispiel Abgabe wurde eingereicht.",
             "saveSuccessful": "Deine Beispiel Abgabe wurde gespeichert.",
-            "readSuccessfully": "Du hast die Bewertungsanweisungen gelesen, du kannst nun mit der Überprüfung der Beispiel Abgabe beginnen.",
+            "readSuccessfully": "Du hast der Beispiel Abgabe erfolgreich gelesen und verstanden.",
```
Comment (cremertim) [inline src/main/webapp/i18n/de/exampleSubmission.json:29]: die Beispielabgabe

Comment (Strohgelaender) [inline src/main/webapp/i18n/de/exampleSubmission.json:29]:
```suggestion
            "readSuccessfully": "Du hast die Beispielabgabe erfolgreich gelesen und verstanden.",
```

**Review by Strohgelaender — DISMISSED** [review]
Tested in testing session, apart from the typo mentioned by @cremertim we could not find any issues.

**Review by mallgaier — DISMISSED** [review]
Tested the creation of a modeling task with two example Submissions as an instructor as specified in the testing steps. Creation worked without errors.

**Pull request merged**